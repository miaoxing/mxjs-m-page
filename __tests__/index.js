import Page from '..';
import {render} from '@testing-library/react';

describe('Page', () => {
  test('basic', () => {
    const result = render(<Page>test</Page>);
    expect(result.container).toMatchSnapshot();
  });
});
