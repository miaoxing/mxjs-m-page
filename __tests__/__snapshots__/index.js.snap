// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page basic 1`] = `
<div>
  <taro-view-core
    class="mx-page css-17ragm3-Component"
    style="--mx-colors-gray-50: #fafafa; --mx-colors-gray-100: #f4f4f5; --mx-colors-gray-200: #e4e4e7; --mx-colors-gray-300: #d4d4d8; --mx-colors-gray-400: #a1a1aa; --mx-colors-gray-500: #71717a; --mx-colors-gray-600: #52525b; --mx-colors-gray-700: #3f3f46; --mx-colors-gray-800: #27272a; --mx-colors-gray-900: #18181b; --mx-space-0: 0; --mx-space-1: 4; --mx-space-2: 8; --mx-space-3: 12; --mx-space-4: 16; --mx-space-5: 20; --mx-space-6: 24; --mx-space-7: 28; --mx-space-8: 32; --mx-space-9: 36; --mx-space-10: 40; --mx-space-11: 44; --mx-space-12: 48; --mx-space-14: 56; --mx-space-16: 64; --mx-space-20: 80; --mx-space-24: 96; --mx-space-28: 112; --mx-space-32: 128; --mx-space-36: 144; --mx-space-40: 160; --mx-space-44: 176; --mx-space-48: 192; --mx-space-52: 208; --mx-space-56: 224; --mx-space-60: 240; --mx-space-64: 256; --mx-space-72: 288; --mx-space-80: 320; --mx-space-96: 384; --mx-fontSizes-xs: 12; --mx-fontSizes-sm: 14; --mx-fontSizes-base: 16; --mx-fontSizes-lg: 18; --mx-fontSizes-xl: 20; --mx-fontSizes-2xl: 24; --mx-fontSizes-3xl: 30; --mx-fontSizes-4xl: 36; --mx-fontSizes-5xl: 48; --mx-fontSizes-6xl: 60; --mx-fontSizes-7xl: 72; --mx-fontSizes-8xl: 96; --mx-fontSizes-9xl: 128;"
  >
    test
  </taro-view-core>
</div>
`;
